src/adagrams.js

```diff
@@ -1,15 +1,79 @@
+const letterPool = {
+  'A': 9, 'B': 2, 'C': 2, 'D': 4, 'E': 12, 'F': 2, 'G': 3, 'H': 2, 'I': 9, 
+  'J': 1, 'K': 1, 'L': 4, 'M': 2, 'N': 6, 'O': 8, 'P': 2, 'Q': 1, 'R': 6, 
+  'S': 4, 'T': 6, 'U': 4, 'V': 2, 'W': 2, 'X': 1, 'Y': 2, 'Z': 1
+}
+
+const getAllLetters = () => {
+  let allLetters = [];
+  for (const letter in letterPool) {
+    const freq = letterPool[letter];
+    allLetters.push(...letter.repeat(freq))
```

Comment:
To make the slightly more readable, I break this logic into two different lines:
```suggestion
    const letterList = letter.repeat(freq);
    allLetters.push(...letterList)
```

This more readily communicates the data structure you are constructing then building.